node/src/actors/chain_manager/mod.rs

```diff
@@ -229,7 +229,7 @@ pub struct ChainManager {
     /// List of superblock votes received while we are synchronizing
     temp_superblock_votes: HashSet<SuperBlockVote>,
     /// Commits and reveals to process later
-    temp_commits_and_reveals: Vec<Transaction>,
+    temp_reveals: Vec<Transaction>,
```

Comment:
This can be made `Vec<RevealTransaction>` to enforce the behavior of only storing reveal transactions:
```suggestion
    temp_reveals: Vec<RevealTransaction>,
```